src/QueueFactory.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Minishlink\WebPush;
+
+class QueueFactory
```

Comment:
What is the purpose of this class?

Reply:
Maintain single instances of Queue classes across the application